test/cp.js

```diff
@@ -112,6 +112,32 @@ test('dest already exists', t => {
   t.is(shell.cat('resources/file2').toString(), oldContents);
 });
 
+test('-nR does not overwrite an existing file at the destination', t => {
+  // Create tmp/new/cp/a
+  const dest = `${t.context.tmp}/new/cp`;
```

Comment:
I feel like we should be using `path.join` to join paths everywhere, but its probably not an issue for now.